### PR TITLE
Feat/merkl preset

### PR DESCRIPTION
**Pull request by envin3**
Merkl incentive presets.
**Tests still missing**

### CODE REVIEW

File: src/settings/index.js [hunk 1]
```diff
@@ -23,7 +23,8 @@ const settings = {
     registrationManager: '0x906B067e392e2c5f9E4f101f36C0b8CdA4885EBf',
     feesManager: '0xDf951d2061b12922BFbF22cb17B17f3b39183570',
     pNetworkAssociationGnosisSafeAddress: '0xf1f6568a76559d85cF68E6597fA587544184dD46',
-    pNetworkV2EthereumVaultAddess: '0xe396757ec7e6ac7c8e5abe7285dde47b98f22db8'
+    pNetworkV2EthereumVaultAddess: '0xe396757ec7e6ac7c8e5abe7285dde47b98f22db8',
+    merklDistributionCreatorAddress: '0x8BB4C975Ff3c250e0ceEA271728547f3802B36Fd'
```
Comment (gitmp01) [inline src/settings/index.js:27]: :heavy_check_mark:

File: src/utils/__tests__/createMerklInflation.test.js
Comment (gitmp01) [inline src/utils/__tests__/createMerklInflation.test.js:1]: :+1:

File: src/utils/presets/createMerklIncentive.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        inflationData.rawAmount
+      ])
+    }
+
+    const createDistributionCalldata = {
+      to: settings.contracts.merklDistributionCreatorAddress,
+      calldata: distributionCreator.encodeFunctionData('createDistribution', [payload])
+    }
+
+    return [...inflationProposal, approve, createDistributionCalldata]
```
Comment (gitmp01) [inline src/utils/presets/createMerklIncentive.js:161]: :+1:

**Review by gitmp01 — APPROVED** [review]
LGTM

File: src/components/complex/CreateProposalModal/index.jsx [hunk 3]
```diff
@@ -138,7 +145,7 @@ const CreateProposalModal = ({ show, onClose }) => {
 
   return (
     <Modal show={show} title="Create Proposal" onClose={onClose} size="lg">
-      {hasPermissionOrEnoughBalance && (
+      {!hasPermissionOrEnoughBalance && (
```
Comment (oliviera9) [inline src/components/complex/CreateProposalModal/index.jsx]: How did the dapp run without this modification?

File: src/utils/__tests__/createMerklInflation.test.js
Comment (oliviera9) [inline src/utils/__tests__/createMerklInflation.test.js:1]: I like this

**Review by oliviera9 — COMMENTED** [review]
I just left a comment

File: src/components/complex/CreateProposalModal/index.jsx (hunk 3 above)
Comment (envin3) [reply src/components/complex/CreateProposalModal/index.jsx]: That was pushed by mistake! It should be only local for testing! Thanks!

**Pull request merged**